ecmabot.js

```diff
@@ -16,11 +16,11 @@ var JSBot = function(profile) {
 	this.sandbox = new Sandbox(path.join(__dirname, "ecmabot-utils.js"));
 	this.factoids = new FactoidServer(path.join(__dirname, "ecmabot-factoids.json"));
 	this.caniuse_server = new CanIUseServer;
-	this.executeRegex = /^((?:sm|v8|js|>>?|\|)>)([^>].*)+/;
+	this.executeRegex = /^((?:sm?|v8?|js?|hs?|b|babel|no?d?e?|>>?|>>>>|\|)>)([^>].*)+/;
```

Comment:
what's `h`/`hs`? 

This also looks like it supports `n>`, `no>`, `nod>`, `node>`, `nd>`, `ne>`, `noe>`, etc - i think we want `n|node` instead of `no?d?e?`